app/views/comments/_comment.html.erb

```diff
@@ -15,3 +17,21 @@
   </p>
 
 </div>
+<%end%>
+<%###################################%>
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.